head/head.go

```diff
@@ -83,10 +82,19 @@ func NewHead(ctx context.Context, options ...opts.Option) (*Head, chan Bootstrap
 		return nil, nil, fmt.Errorf("failed to spawn libp2p node: %w", err)
 	}
 
-	dhtNode, err := dht.New(ctx, node, dhtopts.BucketSize(cfg.BucketSize), dhtopts.Datastore(cfg.Datastore), dhtopts.Validator(record.NamespacedValidator{
-		"pk":   record.PublicKeyValidator{},
-		"ipns": ipns.Validator{KeyBook: node.Peerstore()},
-	}))
+	dhtNode, err := dht.New(
+		ctx,
+		node,
+		dht.Mode(dht.ModeServer),
+		dht.BucketSize(cfg.BucketSize),
```

Comment:
This should always be the default (20).

Reply:
...and I see there is no way to change without using your own protocol prefix. I'll remove this option from Hydra entirely.

Reply:
Instead I added a protocol prefix option so that people can run a Hydra on their own DHT and specify the bucket size if they wish.